### DISCUSSION TITLE
C# Autocomplete lamda code syntax error

### DISCUSSION THREAD

**Discussion by TureeZhang** [community/community#37164] [Copilot Conversations]
There is an "=>" auto generate code syntax error in lamda just like below:

**Copilot generate :**

```c#
.Where(x = > x.Id == id)  //  "= >" space between got an Visual Studio syntax error. 
```

**The right code should be:**

```c#
.Where(x => x.Id == id)  // "=>" no space between
```

Hope fix it in the future update, paid for it and use it more than 3 months, awesome tools.
Thank you so much !

**Comment by iamcjwilliams** [comment]
@TureeZhang Thank you for your feedback! 🚀

**Reply by Vana12330** [reply]
.Where(x => x.Id == id)  // "=>" no space between
